modules/light-clients/08-wasm/types/contract_api.go

```diff
@@ -120,5 +120,5 @@ type checkForMisbehaviourResult struct {
 
 type updateStateResult struct {
 	contractResult
-	Heights []exported.Height `json:"heights"`
+	Heights []clienttypes.Height `json:"heights"`
```

Comment:
yeah, I think this is right, I guess we _could_ keep it as an interface and define structs for each different response type we could have and define a custom unmarshaller. It's not entirely clear to me if that would be needed though.

Also noting that `Status` can also be directly set to `string` instead of `exported.Status`.

Reply:
I think better to use the concrete types when serialising json across the vm with std lib. 
Not sure how well `encoding/json` plays with interfaces in general, its not quite like the proto3 json and the interface support with sdk codec